packages/back-end/src/services/features.ts

```diff
@@ -54,11 +78,22 @@ export async function getFeatureDefinitions(
       rules:
         settings.rules
           ?.filter((r) => r.enabled)
-          ?.map((r) => {
+          ?.map(async (r) => {
             const rule: FeatureDefinitionRule = {};
             if (r.condition && r.condition !== "{}") {
               try {
                 rule.condition = JSON.parse(r.condition);
+                for (const attribute in rule.condition) {
```

Comment:
@jdorn Here is my transform code - it works for some use cases, but given users can enter or conditions with the advanced mode, I think the regex option you talked through today would be much better. Here is the code for reference, though. If you have time tomorrow, I'd love to walk through building that regex with you.